app/assets/stylesheets/_global.scss

```diff
@@ -0,0 +1,24 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
```

Comment:
Set this value so the root font size would be 10px given a default browser font-size of 16px - made rem calculation easy for other font-sizes. I understood this was a standard approach. Do you use a different approach?